### PR TITLE
Update System.Text.Encoding.CodePages docs

### PR DESCRIPTION
**Pull request by jkotas**
## Summary

- Delete suggestion to add System.Text.Encoding.CodePage.dll reference. It is not necessary on supported .NET versions.
- Delete "Windows desktop" references. "Windows desktop" is not the official name for .NET Framework, it is just our internal moniker.
- Delete reference to Windows XP while I was on it.

Related to https://github.com/dotnet/runtime/issues/91940

### CODE REVIEW

File: xml/System/Console.xml [hunk 1]
```diff
@@ -190,7 +190,7 @@
 
  If your app depends on specific code page encodings, you can still make them available by doing the following *before* you call any <xref:System.Console> methods:
 
-1.  Add a reference to the System.Text.Encoding.CodePages.dll assembly to your project.
+1.  Add a reference to the [System.Text.Encoding.CodePages package](https://www.nuget.org/packages/System.Text.Encoding.CodePages) to your project.
```
Comment (tarekgh) [inline xml/System/Console.xml:193]: >Add a reference to the [System.Text.Encoding.CodePages package](https://www.nuget.org/packages/System.Text.Encoding.CodePages) to your project. [](http://example.com/codeflow?start=4&length=143)

This package became inbox in .NET 6.0. may be adding a small sentence clarify that. like "This step is not needed if running on .NET 6.0 or later). Or we just delete this step as we don't support pre .NET 6.0 releases anymore?

File: xml/System.Diagnostics/Process.xml [hunk 2]
```diff
@@ -127,7 +127,7 @@
 
  If a <xref:System.Diagnostics.Process> object depends on specific code page encodings, you can still make them available by doing the following *before* you call any <xref:System.Diagnostics.Process> methods:
 
-1.  Add a reference to the System.Text.Encoding.CodePages.dll assembly to your project.
+1.  Add a reference to the [System.Text.Encoding.CodePages package](https://www.nuget.org/packages/System.Text.Encoding.CodePages) to your project.
```
Comment (tarekgh) [inline xml/System.Diagnostics/Process.xml:130]: >  Add a reference to the [System.Text.Encoding.CodePages package](https://www.nuget.org/packages/System.Text.Encoding.CodePages) to your project. [](http://example.com/codeflow?start=2&length=145)

ditto

**Review by tarekgh — APPROVED** [review]
Thanks @jkotas. I left a small comment which applies to multiple places in this PR. LGTM, otherwise.

File: xml/System/Console.xml (hunk 1 above)
Comment (jkotas) [reply xml/System/Console.xml]: I have deleted it.

**Review by tarekgh — APPROVED** [review]

**Review by carlossanlop — APPROVED** [review]

**Pull request merged**